### DISCUSSION TITLE
How to make request-info bot happy

### DISCUSSION THREAD

**Discussion by ismail** [Homebrew/discussions#3371] [Tap maintenance and Homebrew development] [answered]
I created https://github.com/Homebrew/homebrew-core/pull/104031 initially with an empty description, and request-info bot slapped a helpful `needs-response` label. However, adding a description did not get rid of the label, is there another way to make the bot happy?

**Comment by SMillerDev** [comment, marked as answer]
Those labels are only added when the PR is created.
